docs/src/examples/components/Popup/Variations/PopupExampleOffset.shorthand.tsx

```diff
@@ -4,7 +4,8 @@ import { Button, Grid, Popup } from '@stardust-ui/react'
 const renderButton = rotateArrowUp => (
   <Button
     icon={{
-      name: 'arrow circle up',
+      name: 'arrow-up',
+      circular: true,
```

Comment:
`circular` without `bordered` does not do anything visible. I would rather remove `circular`.
Also think about unifying Popup examples to use the same visual representation among examples.

Reply:
In order to unify the Popup examples, I have decided to keep this icon `circular` and `bordered`.